index.html

```diff
@@ -1,26 +1,54 @@
-
-Course Resources Useful tools
-
-<!-- First link -->
-https://loremipsumgenerator.com/generator/?n=1&t=p Lorem ipsum generator
-
-<!-- Second link  -->
-https://micro-editor.github.io/ Micro: Terminal Text Editor HTML
-
-<!-- First Link -->
-http://www.google.com/search?q=html%20is%20awesome Look for HTML
-
-<!-- Second link -->
-https://developer.mozilla.org/en-US/docs/Web/HTML MDN HTML portal
-
-<!-- Third Link -->
-https://developer.mozilla.org/en-US/docs/Web/HTML/Element MDN HTML element reference
-
-<!-- Fourth Link -->
-https://en.wikipedia.org/wiki/List_of_XML_and_HTML_character_entity_references HTML Entity reference table
-
-<!-- Fifth Link -->
-https://validator.w3.org/#validate_by_upload+with_options HTML validator
-
-<!-- Sixth Link -->
-https://developer.mozilla.org/en-US/docs/Learn MDN HTML Tutorial
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Course resources</title>
+</head>
+
+<body>
+
+  <h1>Course resources</h1>
+
+  <h3>Useful tools</h3>
```

Comment:
i would recommend using an `<h2>` instead of `<h3>` to follow a clean document structure. if you just wanted to achive a small font-size, we will learn how to resize the text with css during the course, without needing to choose between h2 or h3 :v: